README.md

```diff
@@ -285,6 +286,7 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Kraken](https://www.kraken.com) [:rocket:](https://www.kraken.com/careers) | Kraken is a cryptocurrency exchange and bank. | `Remote` |
 | [LoanDolphin](https://loandolphin.com.au/) [:rocket:](https://loandolphin.breezy.hr/) | Home loan marketplace. | `Lisboa` `Remote` |
 | [Moonpay](https://www.moonpay.com) [:rocket:](https://www.moonpay.com/careers) | Solutions for payments, smart contracts, and digital asset management. | `Lisboa` `Remote` |
+| [Pleo](https://www.pleo.io/pt) [:rocket:](https://www.pleo.io/en/careers/jobs) | Centralise your business spending for complete financial efficiency and balance the books effortlessly. | `Lisboa` `Remote` |
```

Comment:
The description is a bit long, could you please shorten it?

Reply:
Changed the description, hopefully is more aligned.